ui/video_type_selection.js

```diff
@@ -0,0 +1,200 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+goog.provide('shaka.ui.VideoTypeSelection');
+
+goog.require('shaka.ui.Controls');
+goog.require('shaka.ui.Enums');
+goog.require('shaka.ui.Locales');
+goog.require('shaka.ui.Localization');
+goog.require('shaka.ui.OverflowMenu');
+goog.require('shaka.ui.SettingsMenu');
+goog.require('shaka.ui.Utils');
+goog.require('shaka.util.Dom');
+goog.requireType('shaka.ui.Controls');
+
+/**
+ * @extends {shaka.ui.SettingsMenu}
+ * @final
+ * @export
+ */
+shaka.ui.VideoTypeSelection = class extends shaka.ui.SettingsMenu {
+  /**
+   * @param {!HTMLElement} parent
+   * @param {!shaka.ui.Controls} controls
+   */
+  constructor(parent, controls) {
+    super(parent, controls,
+        shaka.ui.Enums.MaterialDesignSVGIcons.VIDEO_TYPE);
+
+    this.button.classList.add('shaka-playbackrate-button');
+    this.menu.classList.add('shaka-playback-rates');
+    this.button.classList.add('shaka-tooltip-status');
+
+    if (!Array.from(parent.classList).includes('shaka-overflow-menu')) {
```

Comment:
Maybe I'm mssing something but what was the reason for converting it to an array and using `Array#includes` instead of using the `#contains` method on the `classList` directly?